src/ComparePage/ComparePage.tsx

```diff
@@ -0,0 +1,203 @@
+import {
+	Button,
+	Container,
+	makeStyles,
+	MenuItem,
+	Select,
+	Typography,
+} from '@material-ui/core'
+import React, { FC, useCallback, useContext, useEffect, useState } from 'react'
+import { GlobalContext } from '../GlobalContext'
+import { GetUsersEndpointResult } from '../services/ApiService'
+import StatsDisplay from '../StatsDisplay/StatsDisplay'
+import { theme } from '../theme'
+
+const useStyles = makeStyles({
+	displayContainer: () => ({
+		display: 'flex',
+		flexDirection: 'row',
+		justifyContent: 'space-between',
+	}),
+	searchContainer: () => ({
+		display: 'flex',
+		justifyContent: 'space-between',
+		alignItems: 'center',
+	}),
+})
+
+const ComparePage: FC = () => {
+	const { api, setTitle } = useContext(GlobalContext) // for page title and API access
+
+	const classes = useStyles(theme)
+	// List of users on server
+	const [users, setUsers] = useState<GetUsersEndpointResult[]>([])
+	// Users we are comparing
+	const [userA, setUserA] = useState<GetUsersEndpointResult>({
+		accountId: '0',
+		lastUpdated: 0,
+		masteryTotal: 0,
+		name: '0',
+		summonerId: '0',
+	})
+	const [userB, setUserB] = useState({
+		accountId: '0',
+		lastUpdated: 0,
+		masteryTotal: 0,
+		name: '0',
+		summonerId: '0',
+	})
```

Comment:
Let's leave the default empty here

```suggestion
	const [userA, setUserA] = useState<GetUsersEndpointResult>()
	const [userB, setUserB] = useState<GetUsersEndpointResult>()
```